units/systemd-sysext.service.in

```diff
@@ -24,8 +24,8 @@ ConditionDirectoryNotEmpty=|/usr/lib/extensions
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=@rootlibexecdir@/systemd-sysext --merge
-ExecStop=@rootlibexecdir@/systemd-sysext --unmerge
+ExecStart=@rootbindir@/systemd-sysext --merge
+ExecStop=@rootbindir@/systemd-sysext --unmerge
```

Comment:
`s#@rootbindir@/##` and then move units/systemd-sysext.service.in to units/systemd-sysext.service please.

Reply:
Done, thanks